app/src/main/res/layout/session_list_item.xml

```diff
@@ -67,7 +67,7 @@
     <ImageView
         android:id="@+id/session_list_item_without_video_recording_view"
         android:layout_width="wrap_content"
-        android:layout_height="14sp"
```

Comment:
Shouldn't the changes to video recording icons be in the "Updated Video Icons" commit?

Reply:
The Problem wasn't just with the Video Icons but also the Bell Icon on the Main Screen, so there are also [changes](app/src/main/res/layout/session_layout_land.xml) regarding that. I'll move the Changes to 9c5be3a073b4ff888c442e4c24b6c4dfd34a62f5

Reply:
As I don't want to pull those changes out of a commit again and didn't have confirmation from you yet I've put them in a separate commit for now and will squash that into whichever commit you find suitable

Reply:
The layout for the bell icon on the schedule screen seems to be a bit tricky. In the current implementation of your branch the bell icon sometimes gets **cut off**. Here is a few examples (before/after):

![20230120_schedule_screen_alarm_bells_tablet](https://user-images.githubusercontent.com/144518/213722394-ab843254-0d9b-4eef-8db1-2dc633b6563b.png)

I believe this is why formerly the size of the icon was chosen to **depend on the font size**, hence `sp` instead of `dp`. It might be a general deficit of **how the layout** has been done. Do you want to digg into this issue? The alternative would be to extract the changes concerning the bell icon on the session screen from this pull request and handle it **separately**. All other changes seem to be ready to merge as far as I can tell.

Reply:
I don't think it would be possible to extract that changes, as then the Bell would have the same Size-Problem as the Video Icon. I'll look into this issue :+1:

Reply:
:+1: Please do. I am not sure if both are affected since they are defined in different layout files.